docs/de/demos/README.md

```diff
@@ -0,0 +1,14 @@
+---
+home: false
+# layout: BlogHome
+icon: home
+title: Demos
+sidebar: false
+tagLine: 
+# actions:
+#   - text: Demo
+#     link: https://stage.ocelot.social/
+---
+## XXX
+
+XXX
```

Comment:
As long as we do not have content here, this page should not be reachable.

Reply:
Done